plugins/woocommerce/src/Admin/Features/LaunchYourStore.php

```diff
@@ -68,4 +69,28 @@ public function maybe_create_coming_soon_page( $current_screen ) {
 			update_option( $option_name, $page_id_option, true );
 		}
 	}
+
+	/**
+	 * Preload settings for Site Visibility.
+	 *
+	 * @param array $settings settings array.
+	 *
+	 * @return mixed
+	 */
+	public function preload_settings( $settings ) {
+		$current_screen  = get_current_screen();
```

Comment:
Hey folks, I’m having trouble building WC, I'm getting this error
`[01-Apr-2024 11:49:54 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function Automattic\WooCommerce\Admin\Features\get_current_screen() in /Users/thomas/code/tomato/app/public/wp-content/plugins/woocommerce-monorepo/plugins/woocommerce/src/Admin/Features/LaunchYourStore.php:81`

even though I've run composer install. Is it possible this is running before `get_current_screen` is available? On @nielslange 's site he does not experience this despite us rebuilding and re-cloning the repo, checking out the same branch etc.